package.json

```diff
@@ -49,7 +49,7 @@
     "copy-to-clipboard": "3.3.1",
     "deep-diff": "1.0.2",
     "emotion-theming": "10.0.27",
-    "giantswarm": "git+https://github.com/giantswarm/giantswarm-js-client.git",
+    "giantswarm": "git+https://github.com/giantswarm/giantswarm-js-client.git#generate-for-aws-ha-masters",
```

Comment:
Revert before merge?

Reply:
I'm not merging right away.

I'll update this after merging https://github.com/giantswarm/giantswarm-js-client/pull/113. Thank you for the heads up 👍